### ISSUE TITLE
Nothing's happening

### ISSUE THREAD

**Issue by cmfc31** [TeeSeal/coub-dl#3]
Hi, this is my first time using npm. I downloaded ffmpeg and installed it. But when I try to run your example code. The code doesn't do nothing. I don't know what I'm doing wrong.
![screenshot_1](https://user-images.githubusercontent.com/34674157/38463261-c919b256-3ab3-11e8-9a98-0790361e418c.png)

**Comment by TeeSeal** [comment]
If there is no error then I'm pretty sure it worked 🤔 If you don't specify a full path for the output file it will put the file in your current working directory, in your case thats `C:\Users\cmfc3`, which would be your user directory. Try checking that directory to see if there really isn't any file named `out.mp4`.

You could do:
```
cd C:\Users\cmfc3\Desktop
coub-dl -i https://coub.com/view/135nqc -o out.mp4 --loop 3
```

This way you'll change the current working directory to your desktop, and coub-dl will download the file at that location.

**Comment by cmfc31** [comment]
I also tried that. But today gives me this error message. I think must be something bad about my ffmpeg installation.
![image](https://user-images.githubusercontent.com/34674157/38469785-788186c0-3b17-11e8-9f76-8398b5d9f069.png)

EDIT: It seems that my antivirus detected ffmpeg.exe as a malware so it moved it to the virus vault. I already restored it, added it to my whitelist and restarted windows but still doesn't create the file out.mp4 in my desktop. 
![image](https://user-images.githubusercontent.com/34674157/38470239-22bbd834-3b1d-11e8-8a0c-12e53e64cfc6.png)

I tried this command to test ffmpeg, to get the audio of an mp4 video and it works perfectly.
```
ffmpeg -i video.mp4 -vn -ar 44100 -ac 1 -b:a 32k -f mp3 audio.mp3
```

**Comment by TeeSeal** [comment]
Hmm, I'll have to do some testing on windows then. Will get back at you as soon as I figure something out.

**Comment by TeeSeal** [comment]
@cmfc31 This issue is fixed in the latest 1.4.3 release.

Turns out the problem was with the `loop` option. In order to loop the video I need to create a temporary text file and write the path to the video multiple times inside it, and it seems FFmpeg doesn't like Windows path separators in there `\`, only works with `/`. So I had to force it to use `/` in that file alone instead of the separator used by the system. 😓
